src/components/charts/shared/observationsTables/ObservationsTableCell.jsx

```diff
@@ -7,25 +7,18 @@ import StellarValue from '../StellarValue';
 import StellarValueRange from '../StellarValueRange';
 
 class ObservationsTableCell extends React.PureComponent {
-  rangifier(answerRange) {
```

Comment:
replaced this whole "rangifier" and "answerRange" setup that was built around a bunch of conditionsl with a simple accessor type of "range".